File: labs/solidworks/swex/pmpage/controls/tab/Tab.cs
using CodeStack.SwEx.Common.Attributes;
using CodeStack.SwEx.PMPage.Attributes;
using CodeStack.SwEx.Properties;

public class TabDataModel
{
    [Tab]
    [Icon(typeof(Resources), nameof(Resources.OffsetImage))]
    public class TabControl1
    {
        public string Field1 { get; set; }
    }

    public TabControl1 Tab1 { get; set; }

}

File: labs/solidworks/swex/pmpage/controls/tab/Tab.vb
Imports CodeStack.SwEx.Common.Attributes
Imports CodeStack.SwEx.My.Resources
Imports CodeStack.SwEx.PMPage.Attributes

Public Class TabDataModel

	<Tab>
	<Icon(GetType(Resources), NameOf(Resources.OffsetImage))>
	Public Class TabControl1
		Public Property Field1 As String
	End Class

	Public Property Tab1 As TabControl1

End Class
